_delphi_utils_python/.pylintrc

```diff
@@ -1,8 +1,21 @@
-[DESIGN]
 
-min-public-methods=1
+[MESSAGES CONTROL]
 
+disable=logging-format-interpolation,
+    too-many-locals,
+    too-many-arguments,
+    # Allow pytest functions to be part of a class.
+    no-self-use,
+    # Allow pytest classes to have one test.
```

Comment:
this isn't scoped just to the test dir right? so adding these lines to have tests lint correctly potentially allows these into the actual module code?

secondarily, how strongly do we feel we need to lint unit tests?

Reply:
Yes, they would allow it into the module code, but they seem like narrow enough and innocuous enough problems that they could be caught in review.

If one uses an integrated linter in their IDE, it will read from these files and complain about the tests.  FWIW we enforce style the same across implementation and test files.

Reply:
I'm not a huge fan of relying on code review to catch problems that could have been linted, but agree they're pretty innocuous. One option is to add an additional pylintrc to the tests module, so any weird test things can get captured without worrying about affecting the module code. Not sure how clean that is though.

Reply:
I thought about doing this but thought it created a lot of lint files (2 per indicator rather than the necessary 1).  Can do if you think it is worthwhile.

Reply:
Yea that's kind of annoying...I think they're pretty harmless for now, lets just leave as is.